runtime/doc/eval.txt

```diff
@@ -4723,6 +4724,23 @@ input({opts})
 		more information.  Example: >
 			let fname = input("File: ", "", "file")
 <
+		The optional highlight key allows specifying function which 
+		will be used for highlighting user input.  This function 
+		receives user input as its only argument and must return 
+		a list of 3-tuples [hl_start_byte, hl_end_byte + 1, hl_group] 
+		where
+			hl_start_byte is the first highlighted byte,
+			hl_end_byte is the last highlighted byte (+ 1!),
+			hl_group is |:hl| group used for highlighting.
+					      *E5403* *E5404* *E5405* *E5406*
+		Both hl_start_byte and hl_end_byte + 1 must point to the start 
+		of the multibyte character (highlighting must not break 
+		multibyte characters), hl_end_byte + 1 may be equal to the 
+		input length.  Start column must be in range [0, len(input)), 
+		end column must be in range (hl_start_byte, len(input)], 
```

Comment:
My point is mainly of consistency: there is no need to use obscure math notation to save a few words in otherwise spelled out paragraph text (which is what I comment on, and not an equation)